Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(SRCS
 	Host.cpp
 	RenderWidget.cpp
 	Resources.cpp
+	Translate.cpp
```

Comment:
Should I move all the methods from the Translate class into the Translator class?